src/Microsoft.TestPlatform.CrossPlatEngine/TestEngine.cs

```diff
@@ -109,29 +112,13 @@ public ITestExtensionManager GetExtensionManager()
             return this.testExtensionManager ?? (this.testExtensionManager = new TestExtensionManager());
         }
 
-        /// <summary>
-        /// Retrieves the default test host manager for this engine.
-        /// </summary>
-        /// <param name="runConfiguration">
-        /// The run Configuration.
-        /// </param>
-        /// <returns>
-        /// An instance of the test host manager.
-        /// </returns>
-        public ITestRuntimeProvider GetDefaultTestHostManager(RunConfiguration runConfiguration)
+        /// <inheritdoc/>
+        public ITestRuntimeProvider GetDefaultTestHostManager(string runSettingsXml)
```

Comment:
yes, it gets the appropriate runtime provider for current settings, but I let it be in Interface since that would require a lot of test changes as well